dienst2/settings.py

```diff
@@ -144,12 +140,12 @@
     "django_filters",
     "django.contrib.admin",
     "django.contrib.admindocs",
+    "django_select2",
```

Comment:
Is this application always going to run on a single node? `django-select2` needs a cache backend that is consitent between multiple nodes.

Reply:
ah i see. i am not sure about that.

are there any good alternatives to `django_select`?

Reply:
Not to my knowledge. However, the old situation where we converted the default select to a select2 would still work for this case. (Or are we experiencing bad response times for the post page?)

Reply:
i thought https://www.npmjs.com/package/select2 was outdated and not compatible with bootstrap4. i know see that there is (https://github.com/ttskch/select2-bootstrap4-theme) a bootstrap4 theme. will revert changes and use that theme.

Reply:
reverted